packages/react-dom/src/client/__tests__/dangerouslySetInnerHTML-test.js

```diff
@@ -55,6 +55,7 @@ describe('dangerouslySetInnerHTML', () => {
       );
     });
 
+    // @gate !disableIEWorkarounds
     it('sets innerHTML on it', () => {
```

Comment:
Why does this break?

Reply:
This is in a block that emulates the IE behavior.